beater/connbeat.go

```diff
@@ -89,7 +104,7 @@ func (cb *Connbeat) exportConnection(c Connection, localIPs mapset.Set) error {
 		"remote_ip":     c.remoteIp,
 		"remote_port":   c.remotePort,
 		"local_process": processAsMap(c.process),
-		"container_id":  c.containerId,
+		"container":     toMap(containerInfo),
```

Comment:
It might be nice to leave out the "container" key, when there is no containerInfo. (I assume this now gets translated to json with "container": null)